website/docs/faqs/Accounts/cloud-upgrade-instructions.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "How to upgrade a dbt Cloud account"
+id: "cloud-upgrade-instructions"
+description: "Instructions for upgrading a dbt Cloud account after the trial ends."
+---
+
+dbt Cloud offers [several plans](https://www.getdbt.com/pricing/) with different features that meet your needs. This document is for dbt Cloud users and explains how to select a plan in order to continue using dbt Cloud. 
```

Comment:
Maybe `admin` isn't the best word, but I think we should use stronger language than `users` so it's apparent from the start that this isn't a task anyone can do (and lead into the `tip` that comes next).

```suggestion
dbt Cloud offers [several plans](https://www.getdbt.com/pricing/) with different features that meet your needs. This document is for dbt Cloud admins and explains how to select a plan to continue using dbt Cloud after the trial period ends.
```

Reply:
thanks for this and good callout! i removed the 'afer the trial ends' because someone may need this even if they're not on trial. although that was the scenario that inspired sophia to write this doc, a user can change plans if their trial ends, their account is locked due to team plan cc failure, or they simply want to upgrade/downgrade their account.